tests/pkg-rendition-paginated/OPF/xhtml/nav.xhtml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en" lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<title>Test: Rendition set to paginated</title>
```

Comment:
<title> should match <dc:title> please (not that it matters, but consistency is nice).